src/main/java/sammy/task/Event.java

```diff
@@ -0,0 +1,27 @@
+package sammy.task;
+
+import sammy.Task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    private LocalDateTime startTime;
+    private LocalDateTime endTime;
+
+    public Event(String description, String startTime, String endTime) {
+        super(description);
+        DateTimeFormatter inputFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
```

Comment:
Maybe could change to 'inputFormatter' to better align with the Cody Quality: Naming conventions